grafana/app/dashboards/mark.js

```diff
@@ -0,0 +1,196 @@
+/* global _ */
+
+var ARGS;
+var settings = _.extend({
+  device: 'flame-kk',
+  memory: '319',
+  branch: 'master',
+  suite: 'coldlaunch'
+}, ARGS);
+var title = [
+  'Cold Launch: Marks',
+  settings.device,
+  settings.memory + 'MB',
+  settings.branch
+].join(' | ');
+var dashboard = {
+  id: 10,
+  rows: [],
+  title: title,
+  originalTitle: title,
+  tags: [],
+  style: 'dark',
+  timezone: 'browser',
+  editable: true,
+  hideControls: false,
+  sharedCrosshair: false,
+  nav: [
+    {
+      type: 'timepicker',
+      enable: true,
+      status: 'Stable',
+      time_options: [
+        '5m',
+        '15m',
+        '1h',
+        '6h',
+        '12h',
+        '24h',
+        '2d',
+        '7d',
+        '30d'
+      ],
+      refresh_intervals: [
+        '5s',
+        '10s',
+        '30s',
+        '1m',
+        '5m',
+        '15m',
+        '30m',
+        '1h',
+        '2h',
+        '1d'
+      ],
+      now: true,
+      collapse: false,
+      notice: false
+    }
+  ],
+  time: {
+    from: 'now-24h',
+    to: 'now'
+  },
+  templating: {
+    list: []
+  },
+  annotations: {
+    list: []
+  },
+  refresh: false,
+  version: 6,
+  hideAllLegends: false
+};
+var basePanel = {
+  error: false,
+  span: 4,
+  editable: false,
+  type: 'graph',
+  datasource: 'raptor',
```

Comment:
It's it just the name of the data source in grafana:
https://github.com/taskcluster/taskcluster-stats/blob/master/grafana/config.js#L54

So think this is okay...